src/components/builder/values/values-table.tsx

```diff
@@ -31,19 +31,37 @@ export interface ValuesTableProps<S> {
  * component.
  */
 function ValuesTable<S>({name}: ValuesTableProps<S>) {
+  const intl = useIntl();
   const {getFieldProps} = useFormikContext();
   const {value: options = []} = getFieldProps<Option[] | undefined>(name);
 
+  const tooltip = intl.formatMessage({
+    description: "Tooltip for 'values' builder field",
+    defaultMessage: `The values that can be picked for this field. Values are the text
+    that is submitted with the form data. Labels are the text next to radio buttons,
+    checkboxes and options in dropdowns.
+    `,
```

Comment:
The tooltips not behaving ideally is a known issue, it also overflows in other places. We need to fix the tooltip library rather than the content inside the tooltip. See #35.

Adding the param for the component name can be a nice future improvement, let's maybe see how the re-design turns out for that first?

Reply:
> Adding the param for the component name can be a nice future improvement, let's maybe see how the re-design turns out for that first?

Yes this is really a small detail anyway